README.md

```diff
@@ -1,2 +1,36 @@
 # gp-v2-trading-bot
-Script(s) to interact with gp-v2-contracs
+
+[![Node.js CI](https://github.com/gnosis/gp-v2-trading-bot/actions/workflows/CI.yml/badge.svg)](https://github.com/gnosis/gp-v2-trading-bot/actions/workflows/CI.yml)
+
+*Script(s) to interact with gp-v2-contracts*
+
+This repo can be used for educational purposes to bootstrap a programmatic integration or just serve as a playground to do random trades against the protocol.
+
+## Usage
+
+To install dependencies, run
+
+```
+yarn
+```
+
+To run the trade script, you need an account (in form of a mnemonic or private key) with some funds on you target network and potentially an Infura key (unless you are running on xDAI)
+
+```
+export PK=<private key with some funds>
+export INFURA_KEY=<infura key> // Not needed on xDAI
+```
+
+You also need a [@uniswap/token-lists](https://github.com/Uniswap/token-lists) style token list from which a random trade will be determined (e.g. the [Uniswap default list](https://raw.githubusercontent.com/Uniswap/token-lists/master/test/schema/bigexample.tokenlist.json)).
```

Comment:
Maybe this could be the default parameter if none is set?

Reply:
It won't work on xDAI though :-/

Reply:
But we have a default xDAI token list right?

Reply:
The honeyswap list could be a good default

Reply:
The issue is that quite a few tokens in this list actually don't have sufficient liquidity to trade via uniswap. I think I might have to add a bit more sophisticated logic when it comes to picking a random token pair.